lib/auth/2fa.ts

```diff
@@ -1,19 +1,14 @@
 import { NotifyClient } from "notifications-node-client";
 import { logMessage } from "@lib/logger";
 import { generateTokenCode } from "@lib/auth/tokenGenerator";
+
 const TEMPLATE_ID = process.env.TEMPLATE_ID;
 const NOTIFY_API_KEY = process.env.NOTIFY_API_KEY;
 
-export const generateVerificationCode = async () => {
-  // Temporary - for testing purposes
-  // Will create more robust code generation in the future
-  return await generateTokenCode(5);
-};
+export const generateVerificationCode = async () => await generateTokenCode(5);
```

Comment:
```suggestion
export const generateVerificationCode = async () => generateTokenCode(5);
```
Small performance/memory gain by not rewrapping a resolved promise in another promise.
I'll add a eslint rule for this in another PR.